src/pages/clients/show/Client.tsx

```diff
@@ -87,7 +122,54 @@ export function Client() {
             {t('edit_client')}
           </Button>
 
-          <Dropdown className="divide-y" label={t('more_actions')}></Dropdown>
+          <Dropdown className="divide-y" label={t('more_actions')}>
+            <DropdownElement
+              key={'archive'}
+              onClick={() => {
+                const toastId = toast.loading(t('processing'));
```

Comment:
Please extract this into the method. Mix and match between inline functions/extracted methods shouldn't be present.